admin/src/pages/Order/components/OrderDetails/index.js

```diff
@@ -0,0 +1,93 @@
+import React, { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import OrderDetailTable from "./orderDetailTable";
+import { useDispatch, useSelector } from "react-redux";
+import { Steps, Select } from "antd";
+import "antd/dist/antd.css";
+import { getOrderDetailRequest } from "../../../../actions";
+import Layout from "../../../../components/Layout";
+import FormatDate from "../../../../components/UI/FormatDate";
+
+import "./style.scss";
+
+const { Option } = Select;
+const { Step } = Steps;
+
+export default function OrderDetails() {
+  let { orderId } = useParams();
+  const { orderDetails, loading } = useSelector((state) => state.orders);
+  const [currentStep, setCurrentStep] = useState(1);
+
+  const dispatch = useDispatch();
+  const step = (status) => {
+    if (status == "ordered") return 0;
+    if (status == "in_progress") return 1;
+    if (status == "completed") return 2;
+  };
```

Comment:
```suggestion
  const step = {
    ordered: 0,
    in_progress: 1,
    completed: 2,
  };
```
object cho dễ anh nhé, có thể áp dụng với hàm,...